### DISCUSSION TITLE
Sunbird-RC Bootcamp @ IIIT Delhi on 21st July 2023

### DISCUSSION THREAD

**Discussion by srprasanna** [Sunbird-RC/community#769] [Announcements]
Greetings, fabulous folks! 🌟 Let's keep the Sunbird-RC Bootcamp spirit alive virtually, extending the fantastic in-person event held in Delhi on July 21, 2023. During this boot camp, we'll be diving into the exciting [learning path](https://github.com/orgs/Sunbird-RC/discussions/676). Be sure to stay tuned here for updates on the date, participation, and all the exciting details. Introduce yourself in the comments or subscribe to this discussion thread to receive all the latest notifications. 📢🎉

**Comment by srprasanna** [comment]
Here are the pre-requisites.

#### Hardware Prerequisites:
* Mac / Ubuntu 20.04+ (or any Debian distro) / Windows (with WSL2/GitBash)
* RAM: Min 16 GB RAM
* CPU: Min 4 Cores
* Storage: Min 10 GB disk

#### Software Prerequisites:
* Terminal Emulator or Git Bash/WSL2 (Windows)
* Docker
* Docker Compose
* NodeJS
* npm

##### Installation Guide
* Docker and Docker Compose: https://docs.docker.com/desktop/
* NPM and NodeJS: https://nodejs.org/en

**Reply by Rituparna2112** [reply]
Hi

**Comment by ayushranjan442001** [comment]
Hi Prasanna , Glad to be a part of the hands on session

**Comment by snehal0904** [comment]
Hi, Looking forward to install RC

**Comment by arupdhar** [comment]
Hello there

**Comment by krishnakr6** [comment]
Hi, happy to be part of hands on session . Looking forward to experience sunbird RC.
